### PR TITLE
fix(DB/GameObject): Firework Launcher

### PR DESCRIPTION
**Pull request by Stifler82**
## Changes Proposed:
-  Installed SAI in the GameObject: [Firework Launcher](https://wowgaming.altervista.org/aowow/?object=180771)

## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4487
- Closes https://github.com/chromiecraft/chromiecraft/issues/41

## Tests Performed:
- **Video Test:** [Fireworks at Saltheril's Haven](https://youtu.be/9cxXAPc_7W8)
- Tested in game and working
- Tested on Windows 10 (64)

## Known Issues and TODO List:

- [ ]
- [ ] 

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by Stifler82** [comment]
@FrancescoBorzi

File: data/sql/updates/pending_db_world/rev_1613600380697311600.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
+(180771, 1, 1, 0, 60, 0, 100, 0, 1000, 1000, 15000, 15000, 0, 11, 26351, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Firework Launcher - SE: Update + SA: Cast Spell: Rocket BLUE BIG on Self'),
+(180771, 1, 2, 0, 60, 0, 100, 0, 2000, 2000, 15000, 15000, 0, 11, 26352, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Firework Launcher - SE: Update + SA: Cast Spell: Rocket GREEN BIG on Self'),
+(180771, 1, 3, 0, 60, 0, 100, 0, 3000, 3000, 15000, 15000, 0, 11, 26353, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Firework Launcher - SE: Update + SA: Cast Spell: Rocket PURPLE BIG on Self'),
+(180771, 1, 4, 0, 60, 0, 100, 0, 4000, 4000, 15000, 15000, 0, 11, 26354, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Firework Launcher - SE: Update + SA: Cast Spell: Rocket RED BIG on Self'),
+(180771, 1, 5, 0, 60, 0, 100, 0, 5000, 5000, 15000, 15000, 0, 11, 26355, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Firework Launcher - SE: Update + SA: Cast Spell: Rocket WHITE BIG on Self'),
+(180771, 1, 6, 0, 60, 0, 100, 0, 6000, 6000, 15000, 15000, 0, 11, 26356, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Firework Launcher - SE: Update + SA: Cast Spell: Rocket YELLOW BIG on Self');
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1613600380697311600.sql:10]:
```suggestion
(180771, 1, 1, 0, 60, 0, 100, 0, 1000, 1000, 15000, 15000, 0, 11, 26351, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Firework Launcher - On Update - Cast \'Rocket, BLUE BIG\''),
(180771, 1, 2, 0, 60, 0, 100, 0, 2000, 2000, 15000, 15000, 0, 11, 26352, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Firework Launcher - On Update - Cast \'Rocket, GREEN BIG\''),
(180771, 1, 3, 0, 60, 0, 100, 0, 3000, 3000, 15000, 15000, 0, 11, 26353, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Firework Launcher - On Update - Cast \'Rocket, PURPLE BIG\''),
(180771, 1, 4, 0, 60, 0, 100, 0, 4000, 4000, 15000, 15000, 0, 11, 26354, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Firework Launcher - On Update - Cast \'Rocket, RED BIG\''),
(180771, 1, 5, 0, 60, 0, 100, 0, 5000, 5000, 15000, 15000, 0, 11, 26355, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Firework Launcher - On Update - Cast \'Rocket, WHITE BIG\''),
(180771, 1, 6, 0, 60, 0, 100, 0, 6000, 6000, 15000, 15000, 0, 11, 26356, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Firework Launcher - On Update - Cast \'Rocket, YELLOW BIG\'');

```

**Review by Kitzunu — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1613600380697311600.sql (hunk 1 above)
Comment (FrancescoBorzi) [reply data/sql/updates/pending_db_world/rev_1613600380697311600.sql:10]: I also prefer Keira's format for the comments, but this is not critical and I leave it to the author's personal preference.

**Review by FrancescoBorzi — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1613600380697311600.sql (hunk 1 above)
Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1613600380697311600.sql:10]: It needs to be easily read.
What does SA SE mean? Smart event smart action? 
...

Comment (FrancescoBorzi) [reply data/sql/updates/pending_db_world/rev_1613600380697311600.sql]: yes @Kitzunu

**Pull request merged**